src/Symfony/Component/Notifier/Bridge/FakeChat/composer.json

```diff
@@ -24,8 +24,11 @@
         "php": ">=8.1",
         "symfony/http-client": "^5.4|^6.0",
         "symfony/notifier": "^6.2",
-        "symfony/event-dispatcher-contracts": "^2|^3",
-        "symfony/mailer": "^5.4|^6.0"
+        "symfony/event-dispatcher-contracts": "^2|^3"
+    },
+    "require-dev": {
+        "symfony/mailer": "^5.4|^6.0",
```

Comment:
```suggestion
        "symfony/event-dispatcher": "^5.4|^6.0",
```

Reply:
I just applied the suggestions kind of blindly :) But are you actually sure about the removal of `"symfony/mailer"` here? Is it about moving the event-dispatcher to require-dev as well?

Reply:
I just pushed my assumption of what you meant. Please have a look and let me know if that's fine for you 😉